woocommerce/wc-admin
# Allow for 9 items in Marketing recommended lists

Related: https://github.com/Automattic/woocommerce.com/issues/11522

### Description

WCCOM has been asked to add an extra item for MailPoet to the Recommended Extensions list on the Marketing default page. This list (and the one on the Coupons page) is populated from the https://woocommerce.com/wp-json/wccom/marketing-tab/1.1/recommendations.json endpoint. The `getRecommendedPlugins` resolver passes a `per_page` argument of 6, which determines the number of products shown in the list.

This change increases the argument to 9, and adds a MailPoet SVG icon so the product can be properly rendered when it's added to the WCCOM endpoint.

### Accessibility


### Screenshots

<img width="500" alt="image" src="https://user-images.githubusercontent.com/1647564/140138818-7642137c-ee4c-4e84-a3c3-7d62e46b07a2.png">

### Detailed test instructions:

-  Check out this branch.
-  On a WooCommerce.com local dev environment, check out the branch from https://github.com/Automattic/woocommerce.com/pull/11660.
- Apply this [patch](https://github.com/woocommerce/woocommerce-admin/files/7469590/test-get_recommended_plugins.txt) to your WooCommerce Admin code. This points the `get_recommended_plugins` request to the local WCCOM endpoint, and bypasses a transient check.
- View the Marketing page `wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`. The MailPoet item should appear as in the screenshot, and the link should go to the product page.
- Revert the changes in the patch.
